tests/resources/conformance_suites/nl_nt19/br_kvk/2-04-invalid-period.xbrl

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did you mean to commit the taxonomy package files? They should automatically be downloaded and extracted by the test runner at run time.

Reply:
Those are the local conformance suite files. Those extend the provided conformance suite to cover all of the rules we have for KVK.

Reply:
Sorry, I glossed over this too fast.